test/orgasm/module_test.cpp

```
@@ -0,0 +1,40 @@
+  other.types.push_back(Type{.name = "Result", .variants = Vec<Variant>{{.name = "Ok"}}});
+
+  Function fn;
+  fn.name = "make_result";
+  fn.code = {
+      static_cast<uint8_t>(OpCode::CONSTRUCT_TAGGED),
+      0, 0, // type index
+      0, 0, // variant index
+      0, 0, // payload count
+      static_cast<uint8_t>(OpCode::RETURN),
+  };
+  other.functions.push_back(std::move(fn));
```

Cppcheck (2.20.0)

[error] Uninitialized variables

(uninitvar)